src/deploy.sh

```diff
@@ -478,6 +486,23 @@ function modules_install_to()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function will copy and install the compiled dtb files
+# @target Where should the dtb files be installed (1-vm, 2-local or 3-remote)
+function dtb_install()
+{
+  local target="$1"
+  local architecture="${configurations[arch]}"
+
+  case "$target" in
+    1 | 3) # VM_TARGET or REMOTE_TARGET
+      cp2remote "arch/$architecture/boot/dts/" "/boot/" "" '--include="*/" --include="*.dtb*" --exclude="*"'
```

Comment:
You need to check if `"arch/$architecture/boot/dts/"` exists before trying to copy it. Also, what is this `'--include="*/" --include="*.dtb*" --exclude="*"'`? Could you rewrite this function in way that become easier to undertand your intention?